core/tests/webdriverio_utils/ExplorationEditorMainTab.js

```diff
@@ -887,7 +886,9 @@ var ExplorationEditorMainTab = function() {
 
     // Wait for state name container to completely disappear
     // and re-appear again.
-    await waitFor.visibilityOf(
+    // eslint-disable-next-line oppia/e2e-practices
+    await browser.pause(2000);
```

Comment:
Why? This might need a comment too above.

Reply:
We need to use browser.pause() in order to wait for the state name container to disappear as webdriverio checks for its presence even before it disappears.

Reply:
It the max time needed for the disapperance of state container. I tried 1000ms as well but that doesn't worked.